drivers/sensor/bmi160/bmi160_spi.c

```diff
@@ -0,0 +1,71 @@
+/* Bosch BMI160 inertial measurement unit driver
+ *
+ * Copyright (c) 2020 Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Datasheet:
+ * https://ae-bst.resource.bosch.com/media/_tech/media/datasheets/BST-BMI160-DS000.pdf
+ */
+
+#include <init.h>
+#include <drivers/sensor.h>
+#include <sys/byteorder.h>
+#include <kernel.h>
+#include <sys/__assert.h>
+#include <logging/log.h>
+
+#include "bmi160.h"
+
+#define BMI160_CHIP_ID		0xD1
```

Comment:
Document constants, is this a device address? Is it always constant?

Reply:
Its no device address, Deivce ID(kind of fuse value in register) is always constant based on BMX or BMI chip.

Reply:
So then it should be defined where other constants are defined (bmi160.h). As it's a constant value for all devices, it can be used directly in the code.

Reply:
I meant 0xD1 is for all the BMI160 devices and 0xD8 is for all the BMX160 devices, so cannot directly used in code.

Reply:
I mean that as it's not instance specific, there's no need to store it as part of the driver config structure. Similar to Kconfig values.

Reply:
Unless storing in driver config structure we cannot handle if both BMI160 and BMX160 device  used simultaneous in a board. Please correct me if I misunderstood.

Reply:
Ok, I see. Thanks for the clarification.